tyk-hybrid/templates/deployment-gw-repset.yaml

```diff
@@ -63,8 +63,12 @@ spec:
             value: "{{ .Values.gateway.containerPort }}"
           - name: REDIGOCLUSTER_SHARDCOUNT
             value: "{{ .Values.redis.shardCount }}"
-          - name: TYK_GW_STORAGE_HOSTS
+          - name: TYK_GW_STORAGE_ADDRS
```

Comment:
Seems to be missing ` TYK_GW_STORAGE_ENABLECLUSTER` like in the pro chart.

Reply:
Thanks for noticing this, I have added this option now.